### PR TITLE
Add screenshot_url field to scenario schema to support frontend PR

### PR DESCRIPTION
**Pull request by ahmedhamodi**
*NEW* Loom demo: https://www.loom.com/share/7002cf944d8945b083a511569e9a1ffb

Associated frontend PR: https://github.com/uwblueprint/cdc-frontend/pull/105

### CODE REVIEW

File: app/library/postgres.py
Comment (jaydhulia) [inline app/library/postgres.py:227]: This should be scenario, nice copy pasta 😛

File: app/library/postgres.py
Comment (jaydhulia) [inline app/library/postgres.py:233]: IMO `screenshot_url` is a weird name for this. It makes sense for the scene since we are taking a screenshot of the scene and storing it, but in this case it's not really a screenshot. `display_image_url` or something along those lines might be better.

File: app/library/postgres.py
Comment (ahmedhamodi) [reply app/library/postgres.py:227]: shoot LOL

File: app/library/postgres.py
Comment (jaydhulia) [inline app/library/postgres.py:243]: This won't work, the screenshot_url being saved is the full URL, and what you need to delete is only the key of the object. Try looking at `scene` table's screenshot_url vs the `scenario`:
`scene`: images/247bca4e5f3946619e2633243d9f90f2.jpeg
`scenario`: https://dcc-bp-public-dev.s3.us-east-2.amazonaws.com/assets/cd231a6d3603410a908633c01c7065a5

**Review by jaydhulia — CHANGES_REQUESTED** [review]

File: app/library/postgres.py
Comment (ahmedhamodi) [reply app/library/postgres.py:243]: Yeah I noticed in the scene logic it was appending the asset prefix on the backend and not frontend. I guess this is why. Will fix that.

Comment (jaydhulia) [reply app/library/postgres.py:243]: Yeah, when I implemented that endpoint initially (the screenshot), FE didn't have that asset prefix variable. Now it can be prepended in either FE or BE.

Comment (ahmedhamodi) [reply app/library/postgres.py]: Properly set now :)
image

**Review by jaydhulia — APPROVED** [review]

**Review by ahmedhamodi — COMMENTED** [review]
I'll merge this once frontend PR is approved:
https://github.com/uwblueprint/cdc-frontend/pull/105

@jaydhulia when you get a chance!

**Pull request merged**
